libraries/lib-effects/LoadEffects.cpp

```diff
@@ -154,9 +154,9 @@ void BuiltinEffectsModule::AutoRegisterPlugins(PluginManagerInterface & pm)
       if (rediscoverAll ||
          !pm.IsPluginRegistered(path, &pair.second->name.Msgid())
       ){
-         DiscoverPluginsAtPath(path, ignoredErrMsg, [&](PluginProvider *provider, ComponentInterface *ident)
+         DiscoverPluginsAtPath(path, ignoredErrMsg, [&](PluginProvider *provider, ComponentInterface *ident) -> const PluginID&
          {
-            const auto pluginId = PluginManagerInterface::DefaultRegistrationCallback(provider, ident);
+            const auto& pluginId = PluginManagerInterface::DefaultRegistrationCallback(provider, ident);
```

Comment:
Really, the argument expected by DiscoverPluginsAtPath could be changed to a `std::function` returning void.  (But don't). I was surprised no crashed happend because of this error -- but the return value from the callback is always discarded, in all overrides of `DiscoverPluginsAtPath`.